src/components/Pill/PillButton/__tests__/PillButton.spec.tsx

```diff
@@ -64,78 +92,115 @@ describe('<PillButton> component', () => {
         });
 
         it('should render correctly', () => {
-            expect(toJson(wrapper)).toMatchSnapshot();
-            expect(wrapper.find('.PillButton__pill--isActive')).toHaveLength(0);
-            expect(wrapper.find('.PillButton__pill--isOpen')).toHaveLength(1);
+            expect(view.container).toMatchSnapshot();
+            expect(getButtonByName('')).toHaveClass(
+                'PillButton__button PillButton__button--isOpen'
+            );
+            expect(getButtonByName(name)).not.toHaveClass(
+                'PillButton__pill PillButton__pill--isOpen PillButton__pill--isActive'
```

Comment:
s/a please test for the classes you really don't want to be shown

Reply:
:+1: